.gitlab-ci.yml

```diff
@@ -0,0 +1,204 @@
+  tags:
+    - go
+  cache:
+    key: vendor
+    untracked: true
+    policy: pull-push
+    paths:
+      - vendor/
+  script:
+    - env
+
+checks:
+  stage: test
+  tags:
+    - go
+  script:
+    - |
+      go get -u \
+        golang.org/x/lint/golint \
+        golang.org/x/tools/cmd/goimports \
+        github.com/go-swagger/go-swagger/cmd/swagger
+    - bin/check_golint
+    - bin/check_goimports
+    - bin/check_license
+    - bin/check_govet
+    - bin/check_swagger
+
+test:
+  stage: test
+  tags:
+    - go
+  script:
+    - bin/test
+
+test-e2e:
+  stage: test
+  tags:
+    - go
+  script:
+    - bin/test_e2e
+
+package:linux-amd64:
+  stage: build
+  tags:
+    - go
+  script:
+    - BINARY=build/myst/myst_linux_amd64 bin/package_standalone linux amd64
+
+package:linux-arm:
+  stage: build
+  tags:
+    - go
+  script:
+    - BINARY=build/myst/myst_linux_arm bin/package_standalone linux arm
+  only:
+    - master
+
+package:linux-debian-amd64:
+  stage: build
+  tags:
+    - go
+  script:
+    - go get -u github.com/debber/debber-v0.3/cmd/debber
+    - bin/build
+    - BINARY=build/myst/myst bin/package_debian $BUILD_VERSION amd64
+  only:
+    - master
+
+package:linux-debian-arm:
+  stage: build
+  tags:
+    - go
+  script:
+    - go get -u github.com/debber/debber-v0.3/cmd/debber
+    - bin/build_xgo linux/arm
+    - BINARY=build/myst/myst_linux_arm bin/package_debian $BUILD_VERSION armhf
+  only:
+    - master
+
+package:linux-debian-arm-raspberry:
+  stage: build
+  tags:
+    - go
+  script:
+    - go get -u github.com/debber/debber-v0.3/cmd/debber
+    - bin/build_xgo linux/arm
+    - BINARY=build/myst/myst_linux_arm bin/package_debian $BUILD_VERSION armhf
+    - bin/package_raspberry
+  only:
+    - tags
+
+package:osx-amd64:
+  stage: build
+  tags:
+    - go
+  script:
+    - BINARY=build/myst/myst_darwin_amd64 bin/package_standalone darwin amd64
+  only:
+    - master
+
+package:windows-amd64:
+  stage: build
+  tags:
+    - go
+  script:
+    - BINARY=build/myst/myst_windows_amd64.exe bin/package_standalone windows amd64
+  only:
+    - master
+
+package:ios:
+  stage: build
+  tags:
+    - go
+  script:
+    - bin/package_ios amd64
+  only:
+    - master
+
+package:android:
+  stage: build
+  tags:
+    - go
+  script:
+    - bin/package_android amd64
+  only:
+    - master
+
+package:docker-alpine:
+  stage: build
+  tags:
+    - go
+  script:
+    - bin/package_docker
+    - docker save myst:alpine | gzip > myst_alpine.tgz
+  only:
+    - master
+
+package:docker-ubuntu:
+  stage: build
+  tags:
+    - go
+  script:
+    - bin/package_docker_ubuntu $BUILD_VERSION
+    - docker save myst:ubuntu | gzip > myst_ubuntu.tgz
+  only:
+    - master
+
+package:swagger-redoc-docker:
+  stage: build
+  tags:
+    - go
+  script:
+    - go get -u github.com/go-swagger/go-swagger/cmd/swagger/
+    - bin/swagger_generate
+    - bin/package_docker_docs
+    - docker save tequilapi:${BUILD_VERSION} | gzip > tequilapi_redoc.tgz
```

Comment:
Do we need to export docker images if all tasks are same machine now?

Reply:
we have 4 concurrent builders for now (including 1 machine from Antanas)

Reply:
In the next PR i'll change it to artifacts, so that they are properly published. This is interim setup (PR only)